external/CMakeLists.txt

```diff
@@ -26,27 +26,23 @@ function(pop_variable var)
     set(${var} ${val} PARENT_SCOPE)
 endfunction()
 
-if (DEFINED SPIRV-Headers_SOURCE_DIR)
-  # This allows flexible position of the SPIRV-Headers repo.
-  set(SPIRV_HEADER_DIR ${SPIRV-Headers_SOURCE_DIR})
-else()
-  set(SPIRV_HEADER_DIR ${CMAKE_CURRENT_SOURCE_DIR}/spirv-headers)
-endif()
-
-if (IS_DIRECTORY ${SPIRV_HEADER_DIR})
-  # TODO(dneto): We should not be modifying the parent scope.
-  set(SPIRV_HEADER_INCLUDE_DIR ${SPIRV_HEADER_DIR}/include PARENT_SCOPE)
-
-  # Add SPIRV-Headers as a sub-project if it isn't already defined.
-  # Do this so enclosing projects can use SPIRV-Headers_SOURCE_DIR to find
-  # headers to include.
-  if (NOT DEFINED SPIRV-Headers_SOURCE_DIR)
-    add_subdirectory(${SPIRV_HEADER_DIR})
+find_package(SPIRV-Headers QUIET)
```

Comment:
This would also be more convenient for SDK packaging and system package managers.

Reply:
I don't mind adding something that looks for a package, but I do not like the idea of it being the first place to look. This make development hard because if I have to uninstall the package if I want to use the spirv-headers in the source tree. Please make it the last case if all else fails.

Also, do not remove the use of `SPIRV-Headers_SOURCE_DIR`. That is used by other projects that have spirv-tools as a submodule. This allows them to define their own spirv-tools header. Removing it could break many other projects.

Reply:
> Also, do not remove the use of SPIRV-Headers_SOURCE_DIR. That is used by other projects that have spirv-tools as a submodule. This allows them to define their own spirv-tools header. Removing it could break many other projects.

Why not just check for the TARGET `SPIRV-Headers::SPIRV-Headers` instead? Relying on SPIRV-Headers_SOURCE_DIR is relying on an implementation details of SPIRV-Headers (The project name).

Reply:
If we were doing it from scratch, we would probably do that. However, we have a system in place that is used by many other projects that use spirv-tools, and I do not want to break backward compatibility for no good reason.

Reply:
I've done requested changes.